doc/site/content/en/docs/contributing/dev_new_stages.md

```diff
@@ -100,10 +100,168 @@ The following are standards for testing in Lucille:
 ### Extra Connector Resources
 
 ## Developing Indexers
-[TODO]
 
-### Creating an Indexer
+### Project and Package Layout
+
+Create your indexer under:
+
+```
+lucille/lucille-core/src/main/java/com/kmwllc/lucille/indexer/

```

Comment:
this assumes the developer is contributing to lucille core
there are two other places someone might put an Indexer -- in a separate module dedicated to that indexer, like lucille-plugins/lucille-pinecone -- and also in a third party codebase that uses lucille-core as a dependency